Mathlib/RepresentationTheory/Homological/GroupHomology/LowDegree.lean

```diff
@@ -0,0 +1,268 @@
+/-
+Copyright (c) 2025 Amelia Livingston. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Amelia Livingston
+-/
+import Mathlib.RepresentationTheory.Homological.GroupHomology.Basic
+
+/-!
+# The low-degree homology of a `k`-linear `G`-representation
+
+Let `k` be a commutative ring and `G` a group. This file will contain specialised API for
+the cycles and group homology  of a `k`-linear `G`-representation `A` in degrees 0, 1 and 2.
+In `RepresentationTheory/Homological/GroupHomology/Basic.lean`, we define the `n`th group homology
+of `A` to be the homology of a complex `inhomogeneousChains A`, whose objects are
+`(Fin n →₀ G) → A`; this is unnecessarily unwieldy in low degree. Here, meanwhile, we will define
+the one and two cycles and boundaries as submodules of `G →₀ A` and `G × G →₀ A`, and provide maps
+to `H1` and `H2`.
+
+-/
+
+universe v u
+
+noncomputable section
+
+open CategoryTheory Limits Representation Rep Finsupp
+
+variable {k G : Type u} [CommRing k] [Group G] (A : Rep k G)
+
+namespace groupHomology
+
+section Chains
+variable [DecidableEq G]
+
+/-- The 0th object in the complex of inhomogeneous chains of `A : Rep k G` is isomorphic
+to `A` as a `k`-module. -/
+def zeroChainsIso : (inhomogeneousChains A).X 0 ≅ A.V :=
```

Comment:
What do you think of calling this `chainsIso₀`, and the rest has subscripts 1,2,3?

Another minor benefit of using numerals is that all the names will have the same length.